<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>swc_matrixOutput</class>
 <widget class="QWidget" name="swc_matrixOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SWC Matrix Output</string>
  </property>
  <widget class="QTextBrowser" name="mat00">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat01">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat02">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat12">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat21">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mat22">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="matname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
